config/default/LisOauthService.conf.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
```

Comment:
do we need license here?

Reply:
I don't know, looking at default config files in other extensions they mainly don't contain the licence comment (it's a copy of a generated file from global `config` dir)

Reply:
let's add it, it think is safer :)

Reply:
done